cmd/cli/kubectl-kyverno/test/test_command.go

```diff
@@ -398,7 +398,7 @@ func testCommandExecute(dirPath []string, fileName string, gitBranch string, tes
 				continue
 			}
 
-			if file.Name() == fileName {
+			if strings.Contains(file.Name(), fileName) {
```

Comment:
Can you help me understand why not use the exact match?

Reply:
as now we are getting file name as path like **psp-migration/restrict_adding_capabilities/kyverno-test.yaml**.

Reply:
Which one `file.Name()` or `fileName`?

What about the other one?

Reply:
`fileName` should be set to something like `kyverno-test.yaml`. `file.Name()` should be the appropriate file path. @vyankyGH can we instead compare the base path exactly to `filename` instead? 
```suggestion
			if path.Base(file.Name) == fileName {
```

Reply:
other one define here, 
cmd.Flags().StringVarP(&fileName, "file-name", "f", "kyverno-test.yaml", "test filename")

Reply:
yup @sa

> `fileName` should be set to something like `kyverno-test.yaml`. `file.Name()` should be the appropriate file path. @vyankyGH can we instead compare the base path exactly to `filename` instead?

yup.